src/index.js

```diff
@@ -39,6 +39,11 @@ app.use(
   })
 );
 
+// Set up prometheus metrics
+const promBundle = require("express-prom-bundle");
```

Comment:
```suggestion
import promBundle from "express-prom-bundle"
```
Should be able to do something like this. Guessing its not a named export so no curly braces required. Then you can toss this up top. Linter should order it for you.

Reply:
yep, that worked. thanks!